useraccount/internal/adapters/framework/driver/grpc/proto/server.go

```diff
@@ -0,0 +1,18 @@
+package rpc
```

Comment:
I also believed that this file should be inside the folder grpc and not inside the proto subfolder

Reply:
humm.... that's how it is in the video ?